src/js/index.js

```diff
@@ -248,16 +248,60 @@ export const hpe = deepFreeze({
     },
   },
   checkBox: {
-    color: 'selected-text',
-    gap: 'small',
-    check: {
-      radius: '2px',
+    hover: {
+      border: {
+        color: undefined,
+      },
+      background: {
+        color: 'background-contrast',
+      },
     },
     border: {
+      color: 'border',
       width: '1px',
     },
+    check: {
+      radius: '2px',
```

Comment:
I'd like to be using 'xxsmall' here, but we need to make adjustments to our global.edgeSizes

Reply:
I agree I didn't like putting values but there was no global sizes

Reply:
Can you make an issue for "Create Global Edge Sizes for HPE Theme" and link it to this?

Reply:
yes on it! 👍

Reply:
Along those lines
```suggestion
      radius: '2px', // should use 'xxsmall' of global.edgeSizes
```